anax-in-k8s/Dockerfile.ubi.auto-upgrade-cron.amd64

```diff
@@ -14,7 +14,7 @@ RUN  microdnf update  -y libxml2 --nodocs --setopt=install_weak_deps=0 --disable
   && microdnf install -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager ${REQUIRED_RPMS} \
   && microdnf clean all --disableplugin=subscription-manager \
   && rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/yum.* \
-  && curl -4LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl \
+  && curl -4LO https://storage.googleapis.com/kubernetes-release/release/v1.33.0/bin/linux/amd64/kubectl \
```

Comment:
I don't think we want to hardcode a version here that we have to keep updating 

Currently
curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt
return v1.31.0 

but
```
https://dl.k8s.io/release/stable.txt returns v1.33.0
```

so 
```
 curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
 ```
 will return v1.33.0 currently.... 
 
 
I think using a new URL is a better solution